pandas/sparse/tests/test_frame.py

```diff
@@ -784,12 +784,33 @@ def test_cumsum(self):
         tm.assertIsInstance(result, SparseDataFrame)
         tm.assert_frame_equal(result.to_dense(), expected)
 
+    def test_numpy_cumsum(self):
+        result = np.cumsum(self.frame, axis=0)
+        expected = self.frame.to_dense().cumsum()
+        tm.assertIsInstance(result, SparseDataFrame)
+        tm.assert_frame_equal(result.to_dense(), expected)
+
+        msg = "the 'dtype' parameter is not supported"
+        tm.assertRaisesRegexp(ValueError, msg, np.cumsum,
+                              self.frame, dtype=np.int64)
+
+        msg = "the 'out' parameter is not supported"
+        tm.assertRaisesRegexp(ValueError, msg, np.cumsum,
+                              self.frame, out=result)
+
     def _check_all(self, check_func):
         check_func(self.frame, self.orig)
         check_func(self.iframe, self.iorig)
         check_func(self.zframe, self.zorig)
         check_func(self.fill_frame, self.fill_orig)
 
+    def test_numpy_transpose(self):
+        sdf = SparseDataFrame([1, 2, 3], index=[1, 2, 3], columns=['a'])
+        tm.assert_sp_frame_equal(np.transpose(np.transpose(sdf)), sdf)
+
+        msg = "the 'axes' parameter is not supported"
```

Comment:
generally I like to see

```
result = .....
expected = ....
tm.assert.....
```
more readable. of course if result/expected short then might be ok (and in this case
expected == sdf, so not necessary to add that line)